functions/__z.fish

```diff
@@ -1,15 +1,18 @@
 function __z -d "Jump to a recent directory."
     function __print_help -d "Print z help."
-        printf "Usage: $Z_CMD  [-celrth] dir\n\n"
-        printf "         -c --clean    Cleans out $Z_DATA\n"
-        printf "         -d --dir      Opens directory using system file manager.\n"
+        printf "Usage: $Z_CMD  [-celrth] regex1 regex2...\n\n"
+        printf "         -c --clean    Removes folders that no longer exist from $Z_DATA\n"
```

Comment:
We should be consistent between using folders and directories (this is my fault).

Reply:
thanks, good catch.